canonical-websites/www.ubuntu.com
# update /download/cloud to vbt

## Done

* updated /download/cloud to vanilla brochure theme
* minor clean-up of logic in nav-secondary

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/cloud](http://0.0.0.0:8001/download/cloud)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/patterns%20inventory/9.%20Download?preview=download-cloud.png)

## Issue / Card

Fixes #1622
